protocols/autonat/src/v2/client/handler/dial_back.rs

```diff
@@ -44,19 +45,20 @@ impl ConnectionHandler for Handler {
     ) -> Poll<
         ConnectionHandlerEvent<Self::OutboundProtocol, Self::OutboundOpenInfo, Self::ToBehaviour>,
     > {
-        match self.inbound.poll_unpin(cx) {
-            Poll::Ready(Ok(Ok(nonce))) => {
-                Poll::Ready(ConnectionHandlerEvent::NotifyBehaviour(nonce))
+        match self.inbound.poll_next_unpin(cx) {
+            Poll::Pending => Poll::Pending,
+            Poll::Ready(None) => Poll::Pending,
+            Poll::Ready(Some(Err(err))) => {
+                tracing::debug!("Dial back handler timed out with: {err:?}");
```

Comment:
```suggestion
                tracing::debug!("Stream timed out: {err}");
```

Let's display print these errors.

(`dial_back` will already be part of the "module" that is printed with this log.)